actionpack/test/dispatch/host_authorization_test.rb

```diff
@@ -15,7 +15,7 @@ class HostAuthorizationTest < ActionDispatch::IntegrationTest
     assert_match "Blocked host: www.example.com", response.body
   end
 
-  test "passes all requests to if the whitelist is empty" do
+  test "passes all requests to if the permissions are empty" do
```

Comment:
Sorry to nitpick, but this doesn't really make sense to me. Can we go with `"passes all requests if hosts is empty"`, and `"hosts can be a single element"` below?